tests/test_fips.py

```diff
@@ -54,8 +50,52 @@
 RUN /bin/fips-test sha256
 """
 
+CONTAINER_IMAGES = []
+for param in ALL_CONTAINERS:
+    ctr, marks = container_and_marks_from_pytest_param(param)
+    if param in LTSS_BASE_FIPS_CONTAINERS:
+        CONTAINER_IMAGES.append(param)
+    else:
+        CONTAINER_IMAGES.append(
+            pytest.param(
+                ctr,
+                marks=(
+                    marks
+                    + [
+                        pytest.mark.skipif(
+                            not host_fips_enabled(),
+                            reason="The target must run in FIPS mode for the FIPS test suite",
+                        )
+                    ]
+                ),
+                id=param.id,
+            )
+        )
```

Comment:
For readability's sake, I'd suggest to move the `pytest.mark.skipif` out of the loop (and `tox -e format` please):

```suggestion
_non_fips_host_skip_mark = pytest.mark.skipif(not host_fips_enabled(), reason="The target must run in FIPS mode for the FIPS test suite")

CONTAINER_IMAGES = []
for param in ALL_CONTAINERS:
    ctr, marks = container_and_marks_from_pytest_param(param)
    if param in LTSS_BASE_FIPS_CONTAINERS:
        CONTAINER_IMAGES.append(param)
    else:
        CONTAINER_IMAGES.append(pytest.param(ctr, marks=marks + _non_fips_host_skip_mark), id=param.id))
```

Reply:
All changes have been done :)